zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties

```diff
@@ -1353,6 +1353,9 @@ core.api.action.sendRequest.param.followRedirects =
 core.api.action.sendRequest.param.request = 
 core.api.action.setHomeDirectory = 
 core.api.action.setHomeDirectory.param.dir = 
+core.api.action.setLogLevel = Sets the logging level for a given package or class.
```

Comment:
`given logger name` or `given name`, not necessarily a package/class. (Same in other places it says package/class.)

Reply:
ok